docs/deep-dives/mobile/nimbus-cli/50-working-with-files.mdx

```diff
@@ -6,29 +6,33 @@ slug: /nimbus-cli/working-with-files
 
 ## `fetch`
 
-You can download from a server the current list of all experiments:
+You can fetch a set of one or experiment/rollout recipes:
 
 ```sh
-nimbus-cli --app fenix --channel developer fetch archived.json
+nimbus-cli --app fenix --channel developer fetch --output archived.json android-notification-surface-validation
 ```
 
-Use `preview`, `stage` or `stage/preview` to specify a Remote Settings instance.
+You can use `stage`/`production` to specify a Nimbus API instance.
 
 ```sh
-nimbus-cli --app fenix --channel developer fetch --server preview archived.json
+nimbus-cli --app fenix --channel developer fetch --output archived.json android-research-surface-validation stage/crolloutmobile
```

Comment:
`stage/crolloutmobile` is what we're specifying here? Why is is `stage/`, and where are we getting this `/crolloutmobile` from?

Reply:
It might be nice to show a second example here, to say "here it is on stage, and here's the same thing on production"